File: Modul 4/Soal 4/PRAK404-GALIH_AJI_SABDARAYA-2310817210005.c
#include <stdio.h>

int main(){
    int n;
    double a, b;
    while(1){
        printf("1. penjumlahan");
        printf("\n2. Pengurangan");
        printf("\n3. Perkalian");
        printf("\n4. Pembagian");
        printf("\n5. Exit");
        printf("\nMasukkan Pilihan : ");
        scanf("%d", &n);
        if (n == 1){
            printf("Masukkan nilai pertama : ");
            scanf("%lf", &a);
            printf("Masukkan nilai kedua : ");
            scanf("%lf", &b);
            printf("Hasil penjumlahan antara %.2lf dengan %.2lf adalah %.2lf\n\n", a, b, a+b);
        }
        else if (n == 2){
            printf("Masukkan nilai pertama : ");
            scanf("%lf", &a);
            printf("Masukkan nilai kedua : ");
            scanf("%lf", &b);
            printf("Hasil pengurangan antara %.2lf dengan %.2lf adalah %.2lf\n\n", a, b, a-b);
        }
        else if (n == 3){
            printf("Masukkan nilai pertama : ");
            scanf("%lf", &a);
            printf("Masukkan nilai kedua : ");
            scanf("%lf", &b);
            printf("Hasil perkalian antara %.2lf dengan %.2lf adalah %.2lf\n\n", a, b, a*b);
        }
        else if (n == 4){
            printf("Masukkan nilai pertama : ");
            scanf("%lf", &a);
            printf("Masukkan nilai kedua : ");
            scanf("%lf", &b);
            printf("Hasil pembagian antara %.2lf dengan %.2lf adalah %.2lf\n\n", a, b, a/b);
        }
        else if (n < 1 || n > 5){
            printf("Input anda salah, silahkan coba lagi\n\n");
        }
        else if (n == 5){
            printf("Terimakasih, telah menggunakan kalkulator Galih Aji Sabdaraya");
            break;
        }
    }
    return 0;
}

File: Modul 4/Soal 4/PRAK404-GALIH_AJI_SABDARAYA-2310817210005.py
while True:
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    n = int(input("Masukkan Pilihan : "))
    if n == 1:
        a = float(input("Masukkan nilai pertama : "))
        b = float(input("Masukkan nilai kedua : "))
        operation = a + b
        print(f"Hasil Penjumlahan antara {a:.2f} dengan {b:.2f} adalah {operation:.2f}\n")
    elif n == 2:
        a = float(input("Masukkan nilai pertama : "))
        b = float(input("Masukkan nilai kedua : "))
        operation = a - b
        print(f"Hasil Pengurangan antara {a:.2f} dengan {b:.2f} adalah {operation:.2f}\n")
    elif n == 3:
        a = float(input("Masukkan nilai pertama : "))
        b = float(input("Masukkan nilai kedua : "))
        operation = a * b
        print(f"Hasil Perkalian antara {a:.2f} dengan {b:.2f} adalah {operation:.2f}\n")
    elif n == 4:
        a = float(input("Masukkan nilai pertama : "))
        b = float(input("Masukkan nilai kedua : "))
        operation = a / b
        print(f"Hasil Pembagian antara {a:.2f} dengan {b:.2f} adalah {operation:.2f}\n")
    elif n < 1 or n > 5:
        print("Input anda salah, silahkan coba lagi\n")
    elif n == 5:
        print("Terimakasih, telah menggunakan kalkulator Galih Aji Sabdaraya")
        break